account_product_fiscal_classification/tests/tests.py

```diff
@@ -138,3 +144,12 @@ def test_06_unlink_fiscal_classification(self):
             [self.classification_1_id])
         with self.assertRaises(ValidationError):
             classification.unlink()
+
+    def test_07_classification_generate_from_template(self):
```

Comment:
Or probably something related to the class name:
https://github.com/akretion/account-fiscal-rule/blob/akretion-team/fiscal-classification-template/account_product_fiscal_classification/tests/tests.py#L27

"TestSomething" instead of "Tests"